src/config/validate.ts

```diff
@@ -48,3 +50,10 @@ function validateUrlLifetime(urlLifetime: string): void {
 		throw new InvalidConfigError('URL_LIFETIME specified is invalid')
 	}
 }
+
+function validateLevelLog(logLevel: string) {
+	const levelValues = Object.keys(logger.levels.values)
```

Comment:
Are you sure you need Object.keys() and not Object.values() ?

I'm just asking because the variable name is levelValues. Consider changing it to levelNames if the current usage is correct.

Reply:
You can see the full Object [here](https://github.com/pinojs/pino/blob/master/docs/api.md#levels).
I looking for the *keys* of logger.levels.values, because the values of this object it 10, 20, 30 and so on.